ext/ldap/ldap.c

```diff
@@ -1129,14 +1129,14 @@ PHP_FUNCTION(ldap_bind)
 
 	if (ldap_bind_dn != NULL && memchr(ldap_bind_dn, '\0', ldap_bind_dnlen) != NULL) {
 		_set_lderrno(ld->link, LDAP_INVALID_CREDENTIALS);
-		php_error_docref(NULL, E_WARNING, "DN contains a null byte");
-		RETURN_FALSE;
+		zend_argument_type_error(2, "cannot contain any null-bytes");
```

Comment:
I believe the standard error message is `must not contain null bytes`.

Reply:
Ah, true!